Sources/OAuthSwiftClient.swift

```diff
@@ -16,7 +16,7 @@ var OAuthSwiftDataEncoding: String.Encoding = .utf8
 
 open class OAuthSwiftClient: NSObject {
 
-    fileprivate(set) open var credential: OAuthSwiftCredential
+    open var credential: OAuthSwiftCredential
```

Comment:
`fileprivate(set)` has been added to not allow someone to put a credential associated with a version 1 in a client for oauth 2 and vis versa
If you need to remove it, something must change to check that